### ISSUE TITLE
Functioning theme/template/motif system

### ISSUE THREAD

**Issue by kmark** [paragonie/airship#12]
Figured this is worth tracking since it's probably a critical component for any production usage.

**Comment by paragonie-scott** [comment]
**Status update**.

This is what's already done:

* Installing/updating Motifs:
   * With barge, it should be possible to create a skeleton Motif project. (Will let you know tomorrow.)
   * With Keyggdrasil in working order, updates should be deployable.
* Using Motifs:
   * Users can already select their preferred Motif under the Preferences link in the Bridge.
   * Administrators are able to specify the default motif on a per-Cabin basis. (Both Motifs and Gadgets can be "specific to one Cabin" or "universal".)

Once I'm confident that #11 is done, and I do what I outlined in #19 (should be a quick change), this should be closable.

Creating the user interface for finding/installing extensions (as well as an optional lock file feature that either disables installing new extensions or requires a separate password -- as a mitigation for administrator account compromise) is going to be the time-consuming piece between here and `v0.2.0`.

**Comment by paragonie-scott** [comment]
You can now create a Motif through Barge and it will self-update properly. Then users (or admins) can select them on a per-cabin basis. I still need to do some testing on Cabin deployment and the initial install phase, but the relevant functionality for themes/templates is done.
